src/commonMain/kotlin/au/id/micolous/metrodroid/card/felica/FelicaReader.kt

```diff
@@ -130,6 +141,15 @@ object FelicaReader {
 
                 val services = mutableMapOf<Int, FelicaService>()
 
+                if (onlyFirst && systemNumber > 0) {
+                    // Instead, insert a dummy service with no service codes.
+                    Log.d(TAG, "Not reading system code ${systemCode.hexString}: " +
+                        "onlyFirst = true and systemNumber ($systemNumber) > 0")
+
+                    systems[systemCode] = FelicaSystem(services)
```

Comment:
I'd like some kind of indicator in the dumps that this happened as opposed to having the card with empty systems

Reply:
Done, also added this for skipped services.  We have the data but currently discard it.

I've written a prototype of the UI for this, but still needs to use localised strings.

Reply:
I'm pretty sure we could use the "skipped" _service codes_ to infer filetype metadata and handle "overlay" files in future -- but I'll leave that part for another PR.